pyproject.toml

```diff
@@ -4,6 +4,7 @@ lint.select = [
     "F401", "F821", "F823", "F841",
     "PL", "PLR0913","PLR0914", "PLR0916",
     "YTT",
+    "F541",
```

Comment:
Pre-commit found one more place where f-string is w/o placeholder:

```
09:59:35  --- a/sdcm/tester.py
09:59:35  +++ b/sdcm/tester.py
09:59:35  @@ -2961,7 +2961,7 @@ class ClusterTester(db_stats.TestStatsMixin, unittest.TestCase):  # pylint: disa
09:59:35   
09:59:35           self.destroy_credentials()
09:59:35   
09:59:35  -    @silence(name=f"Save node schema", raise_error_event=False)
09:59:35  +    @silence(name="Save node schema", raise_error_event=False)
09:59:35       def save_cqlsh_output_in_file(self, node, cmd: str, log_file: str):
09:59:35           self.log.info("Save command '%s' output in the file. Node %s", cmd, node.name)
```

Reply:
yep, I didn't built it on latest master

now it should be better